### PR TITLE
Cleans up copy pasted…

### PR DESCRIPTION
**Pull request by Jalleo**
...in monkey and human by merging it at carbon/life.dm

### CODE REVIEW

**Comment by Jalleo** [comment]
If I was reading it right when I noticed this this shouldn't of broken anything.

**Comment by duncathan** [comment]
Are there any children of /carbon/ that have something else?

are there even any other children of carbon

**Comment by Jalleo** [comment]
Working fine for humans and monkeys may cause unintended stuff with aliens and brains once back at home.

**Comment by Jalleo** [comment]
Okay a runtime has been found when admins do a direct transform from human to alien along with deleting the old mob

File: code/modules/mob/living/carbon/alien/life.dm [hunk 1]
```diff
@@ -63,4 +63,7 @@
 
 /mob/living/carbon/alien/CheckStamina()
 	setStaminaLoss(max((staminaloss - 2), 0))
+	return
+
+/mob/living/carbon/alien/handle_changeling()
```
Comment (Jalleo) [inline code/modules/mob/living/carbon/alien/life.dm]: Without this a runtime would of occured due to changeling stuff not existing for aliens

File: code/modules/mob/living/carbon/brain/life.dm [hunk 2]
```diff
@@ -144,8 +144,12 @@
 /mob/living/carbon/brain/handle_status_effects()
 	return
 
+/mob/living/carbon/brain/handle_changeling()
```
Comment (Jalleo) [inline code/modules/mob/living/carbon/brain/life.dm]: same with statement above

**Comment by Jalleo** [comment]
This technically makes no difference apart from reducing the amount of lines that exist but someone could one day try and make changelings able to morph into aliens....

**Pull request merged**